M/MurmurHash3/Compat.toml

```diff
@@ -3,3 +3,6 @@ julia = "0.6-0.7"
 
 ["0.2"]
 julia = "1.0-1.3"
+
+["1.0"]
+julia = "1.0-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.